wormhole/module.py

```diff
@@ -90,7 +90,7 @@ async def _message_formatter(self, message: discord.Message) -> str:
         guild_display = str(emoji) if emoji else f"[{guild.name}]"
 
         formatted_message = (
-            f"**{guild_display} {message.author.name}:** {message.content}"
+            f"**{guild_display} {message.author.name}:**\n{message.content}\n"
```

Comment:
There shouldn't be a newline at the end of the content. The newline before the content should be added only when the content starts with markdown that requires new line (I think it's #, ##, >, >>>, ...)